packages/sdks/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -331,8 +331,12 @@ class BaseDescopeWc extends BaseClass {
     this.#componentsContext = { ...this.#componentsContext, ...e.detail };
   }
 
+  get isRestartOnError() {
+    return this.getAttribute('restart-on-error') === 'true';
```

Comment:
We said default should be false, no?
@dorsha @talaharoni

Reply:
Yes, and we need to have good documentation what to do and how to handles these cases.

Reply:
The default value is indeed false
This is not the default value